src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/MethodOrPropertySymbol.cs

```diff
@@ -61,14 +62,25 @@ public TypeArray Params
             {
                 // Should only be set once!
                 _Params = value;
```

Comment:
If it should be only set once, can we `Debug.Assert(value is null)` on set, and `Debug.Assert(value is not null)` on get?

Reply:
Good idea.